src/components/Activities/Activity3/Game/DebuggingResults.tsx

```diff
@@ -36,7 +35,7 @@ function DebuggingResults(): JSX.Element {
     'Go back and improve recommendations': {
       buttonText: 'Change Priorities',
       onClick: improveRecs,
-      daysMin: 3,
+      daysMin: -10,
```

Comment:
has to have minimum number days 1 so that we have days to allocate (it'll cause bugs/not allow continuing past the `TimeAllocation` phase)
```suggestion
      daysMin: 1,
```